events/wc_massacre_shadow_council.txt

```diff
@@ -0,0 +1,479 @@
+namespace = WCMSC
+
+#Becoming Warchief
+narrative_event = {
```

Comment:
WCMSC.1 is too laggy. It tries to fire everyday for every character. I would recommend to fire it like vanilla HF.49451. Fires when e_spain is created.